### PR TITLE
better formatting of ES cluster status

### PR DESCRIPTION
**Pull request by divyum**
improving the documentation of elasticsearch

### CODE REVIEW

File: docs/reference/getting-started.asciidoc [hunk 1]
```diff
@@ -288,7 +288,13 @@ epoch      timestamp cluster       status node.total node.data shards pri relo i
...
 We can see that our cluster named "elasticsearch" is up with a green status.
 
-Whenever we ask for the cluster health, we either get green, yellow, or red. Green means everything is good (cluster is fully functional), yellow means all data is available but some replicas are not yet allocated (cluster is fully functional), and red means some data is not available for whatever reason. Note that even if a cluster is red, it still is partially functional (i.e. it will continue to serve search requests from the available shards) but you will likely need to fix it ASAP since you have missing data.
+Whenever we ask for the cluster health, we either get green, yellow, or red.
+
+    * Green - everything is good (cluster is fully functional)
+    * Yellow - all data is available but some replicas are not yet allocated (cluster is fully functional)
+    * Red - some data is not available for whatever reason (cluster is partially functional)
+
+**Note:** When a cluster is red, it will continue to serve search requests from the available shards but you will likely need to fix it ASAP since you have missing data.
```
Comment (dakrone) [inline docs/reference/getting-started.asciidoc]: I would shy away from using the term "missing data" in this sentence, instead, I think "unassigned shards" would be a better phrase.

**Review by dakrone — CHANGES_REQUESTED** [review]
I left one comment about wording on this, can you change it and then I'll merge this in?

File: docs/reference/getting-started.asciidoc (hunk 1 above)
Comment (divyum) [reply docs/reference/getting-started.asciidoc]: seems logical. Will make the changes. 👍

**Comment by karmi** [comment]
Hi @divyum, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/elasticsearch/pull/26838.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

File: docs/reference/getting-started.asciidoc (hunk 1 above)
Comment (divyum) [reply docs/reference/getting-started.asciidoc]: @dakrone I have made the changes.

**Comment by divyum** [comment]
Hi @karmi, seems like I forgot to change the email in config. Anyway, done the changes.
Thanks 👍

**Review by dakrone — APPROVED** [review]
Thanks @divyum, this looks good, I'll merge it.

**Pull request merged**